src/ECharts/Commands.purs

```diff
@@ -203,6 +218,9 @@ center a = set "center" $ T.pointToForeign a
 radius ∷ ∀ i. T.Radius → DSL (radius ∷ I|i)
 radius a = set "radius" $ T.radiusToForeign a
 
+singleValueRadius ∷ ∀ i. T.SingleValueRadius → DSL (radius ∷ I|i)
+singleValueRadius a = set "radius" $ T.singleValueRadiusToForeign a

```

Comment:
Same goes for this as with `title`/`titles`, unless you can use either `radius` or `singleValueRadius` any place a radius is expected.